src/applications/disability-benefits/new-686/config/chapters/report-dependent-death/dependent-information/dependentInformation.js

```diff
@@ -105,7 +104,8 @@ export const uiSchema = {
     items: {
       dependentType: dependentTypeUiSchema,
       childStatus: childStatusUiSchema,
-      fullName: _.merge(fullNameUI, {
```

Comment:
This may be fine if this is what you want to do, but `_.merge()` will also merge sub-objects, so the functionality is actually different. Constructing a new object literal in conjunction with the object spread operator will override all subsequent sub-objects.

Which is a lot of words to say this:
```js
const original = {
  foo: {
    bar: 'thingy'
  }
};

_.merge(original, {
  foo: {
    newProp: "Hey, I'm new here."
  }
});
/*
  Creates the following:
  {
    foo: {
      bar: 'thingy',
      newProp: "Hey, I'm new here."
    }
  }
*/

{
  ...original,
  foo: {
    newProp: "Hey, I'm new here."
  }
}
/*
  Creates the following (note the missing foo.bar from original)
  {
    foo: {
      newProp: "Hey, I'm new here."
    }
  }
*/
```

Check it out [in a repl](https://repl.it/repls/VastSphericalExpertise)!

Reply:
Good call out. I'm going to make some changes and request re-review. 👍